podcasts/EffectsPlayer.swift

```diff
@@ -211,23 +209,19 @@ class EffectsPlayer: PlaybackProtocol, Hashable {
     func seekTo(_ time: TimeInterval, completion: (() -> Void)?) {
         guard let readOperation = audioReadTask else { return }
 
-        serialSeekQueue.async { [weak self] in
-            guard let self else { return }
-
-            lastSeekTime = max(0.1, time)
-            seeking = true
-            readOperation.seekTo(time, completion: { [weak self] seekedToEnd in
-                if !seekedToEnd {
-                    completion?()
-                } else if !(self?.playBufferManager?.haveNotifiedPlayer.value ?? false) {
-                    self?.playBufferManager?.haveNotifiedPlayer.value = true
-                    FileLog.shared.addMessage("EffectsPlayer seeked passed end of episode, calling finished playing")
-                    PlaybackManager.shared.playerDidFinishPlayingEpisode()
-                }
+        lastSeekTime = max(0.1, time)
```

Comment:
I did some more tests and, based on my findings, we don't need a queue here since the way it works it's already like a queue. So I just removed it.